src/modules/dashboard/components/CoinMachine/CoinMachine.css

```diff
@@ -60,12 +61,14 @@
 
 .sales {
   composes: container;
-  min-height: 100%;
+  margin-bottom: 50px;
```

Comment:
Why 50px? Specs shared by Karol says 30px. And why do we apply on one of the elements only and not on the larger container?

Reply:
Removed it, applying styles only now to main container